addons/mail/__manifest__.py

```diff
@@ -260,8 +260,19 @@
             ('remove', 'web_tour/static/src/js/tour_manager.js'),
             ('remove', 'web_tour/static/src/js/tour_service.js'),
 
-            'mail/static/src/new/public/**/*',
+            'web/static/src/views/fields/formatters.js',
+            'web/static/src/views/fields/file_handler.*',
+            'web/static/src/views/fields/many2many_tags/tags_list.*',
         ],
+        'mail.assets_public': [
+            ('include', 'mail._assets_public_web'),
+            'mail/static/src/model/model_listener.js',
+            'mail/static/src/component_hooks/use_update.js',
```

Comment:
Could just remove single usage of `useUpdate` in `/new`, so we don't need to import old code